doc/ondemand.md

```diff
@@ -421,30 +453,46 @@ mutable reference to it.
 
 ### Iteration Safety
 
-  - If the value fails to be parsed as one type, you can try to parse it as something else until you
```

Comment:
Thanks for trying to translate my incomplete sentences and half thoughts into reality :) The second half of this document was really kind of "draft information" that hadn't been turned into a full document.